tests/lib/rules/no-namespace.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @fileoverview Disallows the use of internal modules and namespaces.
```

Comment:
Again please could we amend the use of "internal vs external" in this file to something more TypeScript specific. I do not feel it is clear enough what internal and external denote.